src/dodal/beamlines/b01-1.py

```diff
@@ -0,0 +1,67 @@
+from ophyd_async.core import StaticDirectoryProvider
+from ophyd_async.epics.areadetector import AravisDetector
+from ophyd_async.panda import HDFPanda
+
+from dodal.common.beamlines.beamline_utils import (
+    device_instantiation,
+)
+from dodal.common.beamlines.beamline_utils import set_beamline as set_utils_beamline
+from dodal.log import set_beamline as set_log_beamline
+
+# BL = get_beamline_name("BL01C")
+BL = "c01"
+set_log_beamline(BL)
+set_utils_beamline(BL)
+
+
+static_directory_provider = StaticDirectoryProvider("/tmp/bluesky_test_static")
+# set_directory_provider(PandASubdirectoryProvider())
+
```

Comment:
```suggestion
set_directory_provider(StaticDirectoryProvider("/dls/b01-1/data/"))
```

File system seems to be hiccuping (again) but when it's recovered find a sensible visit dir for this beamline

Reply:
is the file system gaslighting me at the moment?

![image](https://github.com/user-attachments/assets/5c7c22a0-cb47-41ec-81fe-94d3ae865c55)

Reply:
I think someone needs to raise a ticket because it's been consistently like that for days.

Reply:
made a ticket https://jira.diamond.ac.uk/servicedesk/customer/portal/4/ITHD-4851